spec/index.bs

```diff
@@ -7249,6 +7277,18 @@ enum GPUQueryType {
 };
 </script>
 
+## Occlusion Query ## {#occlusion}
+
+Occlusion query is only available on render pass, to query the number of samples that pass all the
+per-fragment tests for a set of drawing commands, including scissor, sample mask, alpha to
+coverage, stencil, and depth tests. Any non-zero result value for the query indicates that at least
+one sample passes the tests, 0 indicates that no sample passes the tests.
```

Comment:
"one sample passes the tests" -> "one sample passed the tests and reached the output merging stage of the pipeline"